.prettierrc

```diff
@@ -6,5 +6,13 @@
   "printWidth": 80,
   "tabWidth": 2,
   "useTabs": false,
-  "proseWrap": "always"
+  "proseWrap": "always",
+  "overrides": [
+    {
+      "files": "*.css",
+      "options": {
+        "singleQuote": false
```

Comment:
We should look into adding this to dvc.org!

Reply:
dvc.org took a different approach by changing `format-staged` not to target css files, but giving it this configuration will probably make linting more robust.